packages/zoe/src/zoeService/types.js

```diff
@@ -181,14 +181,40 @@
  */
 
 /**
+ * Zoe uses seats to access or manipulate offers. They let contracts and users
+ * interact with them. Zoe has two kinds of seats. ZCFSeats are used within
```

Comment:
Which of They/them refers to seats vs. offers?

Where did this text come from? Ah... `@link`. cool.